app/helpers/application_helper/button/generic_object_definition_delete_button.rb

```diff
@@ -0,0 +1,17 @@
+class ApplicationHelper::Button::GenericObjectDefinitionDeleteButton < ApplicationHelper::Button::Basic
+  def visible?
```

Comment:
def visible?
  @display != 'generic_objects'
end

Reply:
Done in 446f7c4